sss.asado-analysis/src/main/scala/sss/analysis/Main.scala

```diff
@@ -28,7 +28,7 @@ object Main {
     clientNode.actorSystem.actorOf(Props(classOf[AnalysingActor], clientNode).withDispatcher("my-pinned-dispatcher"))
     clientNode.initStateMachine
 
-
+    // wouldn't pure config lib be a cleaner solution (ie reduce code size)
```

Comment:
Tell me more... ?

Reply:
it will automatically couple the case class val names and keys in .conf
So you don't need to do it manually.

For example

lala.hoho-mama = true

case class Config(lala: HohoMamaConf)
case class HohoMamaConf(hohoMama: Boolean)

This will be magically mapped